js/common/lib/inference-session.ts

```diff
@@ -243,7 +243,7 @@ export declare namespace InferenceSession {
   }
   export interface WebNNExecutionProviderOption extends ExecutionProviderOption {
     readonly name: 'webnn';
-    deviceType?: 'cpu'|'gpu';
+    deviceType?: 'cpu'|'gpu'|'npu';
```

Comment:
We may need another parameter in the future like a fallback device type, or to let this be a *list* of devices in priority order (as a hint/preference anyway), but this lets us at least *test* it within ORT.